articles/backup/backup-mabs-files-applications-azure-stack.md

```diff
@@ -128,5 +128,10 @@ To view Azure Backup Server entities in the Azure portal, you can follow the fol
 
 For information on using Azure Backup Server to protect other workloads, see one of the following articles:
 
+* [What is Azure Stack Hub](./azure-stack-overview.md)
+* [What is Azure Backup Service](./backup-overview.md)
+* [What is Azure AD](./active-directory/fundamentals/active-directory-whatis.md)
+* [What is Azure Recovery Services Vault](./backup-azure-recovery-services-vault-overview.md)
+* [What is Azure Storage](./storage-introduction.md)
```

Comment:
This link is incorrect. Please confirm if you want to provide the [link to this article](https://docs.microsoft.com/azure/storage/common/storage-introduction).

Reply:
Hi, pls find the updated links:

[What is Azure Backup Service] (./backup/backup-overview.md)
[What is Azure AD]  (./active-directory/fundamentals/active-directory-whatis.md)
[What is Azure Recovery Services Vault] (./backup/backup-azure-recovery-services-vault-overview.md)
[What is Azure Storage] (./storage/common/storage-introduction.md)